scoreboard/app.py

```diff
@@ -66,6 +67,10 @@ def create_app(config=None):
         app.logger.handlers[0].setFormatter(log_formatter)
         logging.getLogger().handlers[0].setFormatter(log_formatter)
 
+    #Scss is expecting this
+    app.debug = app.config.get('DEBUG')
+    scss_compiler = Scss(app, static_dir='static/css', asset_dir='static/scss')
+    scss_compiler.update_scss()
```

Comment:
Appengine filesystem is read only. Entire compilation process won't work as
a result. Appengine will need a pre-compilation step before deploy

On Thu, Jun 9, 2016 at 8:56 AM, Zach Wade <notifications@github.com> wrote:

> In scoreboard/app.py
> <https://github.com/google/ctfscoreboard/pull/29#discussion_r66468882>:
>
> > @@ -66,6 +67,10 @@ def create_app(config=None):
> >          app.logger.handlers[0].setFormatter(log_formatter)
> >          logging.getLogger().handlers[0].setFormatter(log_formatter)
> >
> > +    #Scss is expecting this
> > +    app.debug = app.config.get('DEBUG')
> > +    scss_compiler = Scss(app, static_dir='static/css', asset_dir='static/scss')
> > +    scss_compiler.update_scss()
>
> What specifically won't work. The debug portion, or the entire compilation
> process?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/ctfscoreboard/pull/29/files/ff7f0d2ecc4fa9a6faf7a7617877a24bdc3d0c8e#r66468882>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKVZCQdzitFjryT7GIdygJEK0h49KvuRks5qKDfCgaJpZM4IxiYR>
> .
>